### PR TITLE
Make SupportView login_required

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: readthedocs/urls.py [hunk 1]
```diff
@@ -26,7 +27,7 @@
         TemplateView
         .as_view(template_name='security.txt', content_type='text/plain'),
     ),
-    url(r'^support/$', SupportView.as_view(), name='support'),
+    url(r'^support/$', login_required(SupportView.as_view()), name='support'),
```
Comment (stsewd) [inline readthedocs/urls.py:30]: We use this mxin in our code base https://github.com/readthedocs/readthedocs.org/blob/3c9dafe093ff444fa304f660b661ad0be3e6316f/readthedocs/core/mixins.py#L20-L20

Comment (ericholscher) [reply readthedocs/urls.py]: Gotcha, should probably chagne https://github.com/readthedocs/readthedocs.org/blob/1758dc9234ce18e0fcb4076fd37a400724e99951/readthedocs/projects/urls/private.py#L66-L65 as well.

Comment (ericholscher) [reply readthedocs/urls.py]: Looks like Django has this builtin now: https://docs.djangoproject.com/en/dev/topics/auth/default/#django.contrib.auth.mixins.LoginRequiredMixin

Comment (stsewd) [reply readthedocs/urls.py]: yeah, our mixin basically inherits from that, I think we can use django's mixin directly, don't think we would be adding any other logic there.

**Review by stsewd — APPROVED** [review]

**Pull request merged**